src/app/examples/doughnut-chart-ns-example/doughnut-chart-ns-example.component.ts

```diff
@@ -16,25 +19,16 @@ export class Distribution {
 })
 
 export class DoughnutChartNsExampleComponent implements OnInit {
-  private _distributions$: Observable<Distribution[]>;
-  private _chartModels$: Observable<ChartModel[]>;
+  private chartModels$: Observable<ChartModel[]>;
   private _paletteColors = ['#33a87f', '#005d3c', '#02f5a1', '#24765a'];
-  buttonText: string;
-  title: string;
-  subtitle: string;
+  buttonText = 'Se mere';
+  title = 'Investeringsoverblik';
+  subtitle = '';
 
   constructor() { }
 
   ngOnInit() {
-    this.buttonText = 'Se mere';
-    this.title = 'Investeringsoverblik';
-    this.subtitle = '';
-    this._distributions$ = of(this.getDistributionsMock()).pipe(delay(1000));
-    this._chartModels$ = this.convertToChartModel(this._distributions$, this._paletteColors);
-  }
-
-  get chartModels$() {
-    return this._chartModels$;
+    this.chartModels$ = this.convertToChartModel(of(this.getDistributionsMock()).pipe(delay(1000)), this._paletteColors);
   }
 
   getDistributionsMock(): Distribution[] {
```

Comment:
Still think this needs to be renamed to `getMockData` or similar. Also consider changing the mock data to something non-banking specific. This is Kirby, not DRB... :)

Reply:
Done.